ProjectLighthouse.Servers.Website/Pages/NotificationsPage.cshtml.cs

```diff
@@ -0,0 +1,113 @@
+﻿using LBPUnion.ProjectLighthouse.Configuration;
+using LBPUnion.ProjectLighthouse.Database;
+using LBPUnion.ProjectLighthouse.Helpers;
+using LBPUnion.ProjectLighthouse.Servers.Website.Pages.Layouts;
+using LBPUnion.ProjectLighthouse.Types.Entities.Notifications;
+using LBPUnion.ProjectLighthouse.Types.Entities.Profile;
+using LBPUnion.ProjectLighthouse.Types.Entities.Website;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace LBPUnion.ProjectLighthouse.Servers.Website.Pages;
+
+public class NotificationsPage : BaseLayout
+{
+    public NotificationsPage(DatabaseContext database) : base(database)
+    { }
+
+    public List<WebsiteAnnouncementEntity> Announcements { get; set; } = new();
+    public List<NotificationEntity>? Notifications { get; set; } = new();
+    public string Error { get; set; } = "";
+
+    public async Task<IActionResult> OnGet()
+    {
+        this.Announcements = await this.Database.WebsiteAnnouncements
+            .Include(a => a.Publisher)
+            .OrderByDescending(a => a.AnnouncementId)
+            .ToListAsync();
+
+        if (this.User == null) return this.Page();
+
+        this.Notifications = await this.Database.Notifications
+            .Where(n => n.UserId == this.User.UserId)
+            .Where(n => n.IsDismissed == false)
+            .Include(n => n.User)
```

Comment:
```suggestion
```
I don't think you're actually using the user object of the notification so this can be safely removed